pkg/rollout/rollout.go

```diff
@@ -304,8 +304,7 @@ func (r *Rollout) updateAnnotations(svc *run.Service, stable, candidate string)
 
 // diagnoseCandidate returns the candidate's diagnosis based on metrics.
 func (r *Rollout) diagnoseCandidate(candidate string, healthCriteria []config.Metric) (d health.Diagnosis, err error) {
-	// TODO(gvso): Consider using a different config value for the offset.
-	healthCheckOffset := time.Duration(r.strategy.Interval) * time.Second
+	healthCheckOffset := time.Duration(r.strategy.HealthIntervalMinute) * time.Minute
```

Comment:
might be wise to accept `int` or `int64` for these types (instead of uint64).

I believe that way we can directly multiply e.g. `:= var * time.Minute`. though needs double-checking.